daml/Demo.daml

```diff
@@ -79,7 +79,7 @@ examples Providers{..} Assets{..} issuer = do
   (_, del)    <- originate "DEL-TSLA" "Delivery Example" $ delivery tsla.assetId 20.0
   (_, dvp)    <- originate "DVP-TSLA-USD" "DvP Example" $ dvp tsla.assetId 20.0 usd.assetId 10_000.0
   (_, fwd)    <- originate "FWD-TSLA-USD-20221022" "Forward Example" $ futureDvp tsla.assetId 20.0 usd.assetId 10_000.0 (date 2022 Oct 22)
-  (_, bnd)    <- originate "BOND-USD-4.875" "Forward Example" $ bond usd.assetId 1000000.0 0.04875 $ unrollDates 2021 2023 [Jan,Jul] 15
+  (_, bnd)    <- originate "BOND-USD-4.875" "Bond Example" $ bond usd.assetId 1000000.0 0.04875 $ unrollDates 2021 2023 [Jan,Jul] 15
```

Comment:
The repeated value of `"Forward Example"` here actually resulted in a runtime React error, because the asset dropdowns/selections use these labels as the keys of the components being iterated out on. 🤔 

this was obviously a typo, but we may need to think about how to better support having multiple assets with the same name in the UI, _or_ enforce asset name uniqueness in the Daml models

Reply:
`AssetDescription` is keyed on the `assetId` which is the label and the signatories from the `Issuance` service `[provider, customer]`...so there is name uniqueness per `Issuance` service. So at least for the keys, if you keyed the dropdowns as `${assetId.label}-${assetId.signatories}`, that should work.